src/current/_includes/releases/v23.1/v23.1.17.md

```diff
@@ -0,0 +1,103 @@
+## v23.1.17
+
+Release Date: March 19, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-17-security-updates">Security updates</h3>
+
+- A user with the `VIEWACTIVITY` privilege can now request a statement bundle from the [**Statements** page](https://www.cockroachlabs.com/docs/v23.1/ui-statements-page) in the DB Console or with the internal builtin function `crdb_internal.requests_statement_bundle`. [#119638][#119638]
```

Comment:
'the VIEWACTIVITY privilege' could be a link to https://www.cockroachlabs.com/docs/v23.1/security-reference/authorization#supported-privileges